cmd/crio/main.go

```diff
@@ -59,15 +62,18 @@ func validateConfig(config *server.Config) error {
 func mergeConfig(config *server.Config, ctx *cli.Context) error {
 	// Don't parse the config if the user explicitly set it to "".
 	if path := ctx.GlobalString("config"); path != "" {
+		// Use the build-time-defined defaults path
+		if DefaultsPath != "" {
+			path = DefaultsPath + "/crio.conf"
```

Comment:
will this override what the user sets for `--config=$PATH`?  The CLI setting needs to have the highest precedence.

Also, please use `filepath.Join` instead of string concatenation.